### PR TITLE
Leaderboard: hit the API

### PR DESCRIPTION
**Pull request by VelzckC0D3**
# Leaderboard API -  `WebPack` `ES6` `API`

## ✨ _Requested Implementations_ ✨
✔️ _Created a new game with a personalized ID_
✔️ _Imeplemented refresh button that gets all scores from the personalized API_
✔️ _Implemented submit button which uploads the .JSON new scores to the API_
✔️ _ES6 Used (arrow functions included)_
✔️ _Used ASYNC and Await_
✔️ _No style added_

#  💡 _Thank you for your review!_ 💡

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -1,5 +1,6 @@
 import './style.css';
 
+// Generate HTML
 function component() {
   const element = document.createElement('section');
```
Comment (AnselemOdims) [inline src/index.js]: - [ ] Good job working on this, but please kindly ensure that you use ES6 arrow functions.

**Review by AnselemOdims — CHANGES_REQUESTED** [review]
### Hi @VelzckC0D3 , 
You have been doing a tremendous job so far :ok_hand:
There are some issues that you still need to work on to go to the next project but you are almost there!

## To Highlight 🎉
PR has a title and summary. :heavy_check_mark:
No linter errors :heavy_check_mark:

## Required Changes ♻️
- [ ] Please also check inline comments under the review

## Optional suggestions
_Comments with the [OPTIONAL] prefix are not a must to implement however, It's strongly recommended to work on them as they can make your code better._

## Happy coding!👏👏👏

#### Please leave any questions or comments in the PR if you have any doubts or unclarity! You can tag me using @AnselemOdims for a quicker response

**_Note: Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

File: src/index.js (hunk 1 above)
Comment (VelzckC0D3) [reply src/index.js]: @AnselemOdims Oh! I missed that one! thank you for noticing hahaha, I'll correct it right away 
image

**Review by amejid — APPROVED** [review]
Hi @VelzckC0D3, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights
- Linter checks passing :heavy_check_mark: 
- Correct use of Gitflow :heavy_check_mark: 
- API calls working as expected :heavy_check_mark: 
- ES6 syntax used :heavy_check_mark: 
## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**